### PR TITLE
Fix broken link

### PR DESCRIPTION
**Pull request by jonjohnsonjr**

### CODE REVIEW

File: build/builds.md [hunk 1]
```diff
@@ -156,7 +156,7 @@ complement the volumes that are implicitly
 
 For example, use volumes to accomplish one of the following common tasks:
 
- * [Mount a Kubernetes secrets(./auth.md).
+ * [Mount a Kubernetes secret](./auth.md).
 
  * Creat an `emptyDir` volume to act as a cache for use across multiple build
```
Comment (ImJasonH) [inline build/builds.md]: * Create

Comment (jonjohnsonjr) [reply build/builds.md]: > Ken Thompson was once asked what he would do differently if he were redesigning the UNIX system. His reply: "I'd spell creat with an e."

**Pull request merged**